client/src/pages/about/AboutPresident.tsx

```diff
@@ -13,7 +14,10 @@ interface AboutPresidentProps {
 const UnstyledAboutPresident: FunctionComponent<AboutPresidentProps> = ({
   className,
 }) => {
-  return <Box>President page</Box>;
+  return (
+  <>
+  <President/>
+  </>);
```

Comment:
This component is not doing anything than just exporting out <President> comp. And looks like this is not even used anywhere else? what's the need for hits `AboutPresidnet` component?